### PR TITLE
18NewEngland initial config/game

### PR DESCRIPTION
**Pull request by scottredracecar**

### CODE REVIEW

File: lib/engine/config/game/g_18_newengland.rb [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+    "4": 16,
+    "5": 13
+  },
+  "startingCash": {
+    "2": 600,
+    "3": 400,
+    "4": 280,
+    "5": 280
+  },
+  "capitalization": "inceremental",
```
Comment (tobymao) [inline lib/engine/config/game/g_18_newengland.rb:29]: spelled wrong

File: lib/engine/game/g_18_newengland.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                      lightBlue: '#a2dced',
+                      yellow: '#FFF500',
+                      orange: '#f48221',
+                      brown: '#7b352a')
+
+      load_from_json(Config::Game::G18NewEngland::JSON)
+
+      DEV_STAGE = :beta
+
+      def action_processed(action)
```
Comment (tobymao) [inline lib/engine/game/g_18_newengland.rb:21]: delete evrything here

File: lib/engine/game/g_18_newengland.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                      red: '#d81e3e',
+                      blue: '#0189d1',
+                      lightBlue: '#a2dced',
+                      yellow: '#FFF500',
+                      orange: '#f48221',
+                      brown: '#7b352a')
+
+      load_from_json(Config::Game::G18NewEngland::JSON)
+
+      DEV_STAGE = :beta
```
Comment (tobymao) [inline lib/engine/game/g_18_newengland.rb:19]: it's not beta

File: lib/engine/config/game/g_18_newengland.rb (hunk 1 above)
Comment (scottredracecar) [reply lib/engine/config/game/g_18_newengland.rb]: fixed in new commit, is this the proper way to do it?

File: lib/engine/game/g_18_newengland.rb (hunk 2 above)
Comment (scottredracecar) [reply lib/engine/game/g_18_newengland.rb]: fixed

File: lib/engine/game/g_18_newengland.rb (hunk 3 above)
Comment (scottredracecar) [reply lib/engine/game/g_18_newengland.rb]: fixed

**Pull request merged**